### PR TITLE
Add build with extensions info and grammar corrections

### PR DESCRIPTION
**Pull request by nebraszka**

### CODE REVIEW

File: include/rgl/api/core.h [hunk 1]
```diff
@@ -167,7 +167,7 @@ typedef enum : int32_t
 	RGL_INITIALIZATION_ERROR,
 
 	/**
-	 * Requested functionality has been not yet implemented.
+	 * Requested functionality still needs to be implemented.
```
Comment (PiotrMrozik) [inline include/rgl/api/core.h:170]: problem -> opportunity  :upside_down_face:

File: include/rgl/api/core.h [hunk 2]
```diff
@@ -275,7 +275,7 @@ rgl_cleanup(void);
 /******************************** MESH ********************************/
 
 /**
- * Creates mesh from vertex and index arrays. CW/CCW order does not matter.
+ * Creates mesh from a vertex and index arrays. CW/CCW order does not matter.
```
Comment (PiotrMrozik) [inline include/rgl/api/core.h:278]: To avoid the confusion of plurals:
"Creates mesh from the arrays of vertices and indices."

**Review by PiotrMrozik — COMMENTED** [review]
It was *THE* code review :stuck_out_tongue_winking_eye:

File: docs/GaussianNoise.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Gaussian noise
 
-Robotec GPU Lidar (RGL) allows introducing Gaussian noise which models the naturally present noise of the physical devices.
+Robotec GPU Lidar (RGL) allows the introduction of Gaussian noise, which models the naturally present noise of the physical devices.
```
Comment (prybicki) [inline docs/GaussianNoise.md:3]: Hmm

File: docs/GaussianNoise.md [hunk 4]
```diff
@@ -12,14 +12,14 @@ It is parametrized by these two values:
 
 ## Supported noise types
     
-The following table presents concisely all kinds of Gaussian noise available in RGL.
-The next chapters describe them in greater detail.
+The following table presents all kinds of Gaussian noise available in RGL concisely.
+The subsequent chapters describe them in greater detail.
```
Comment (prybicki) [inline docs/GaussianNoise.md:16]: Hmm

File: docs/Usage.md [hunk 5]
```diff
@@ -1,34 +1,34 @@
...
-The coordinate system in RGL is right-handed: the positive x and y axes point right and up, and the negative z axis points forward. Positive rotation is counterclockwise about the axis of rotation.
-When created, entity is bound to a scene. Entity cannot be unbound from a scene or bound to multiple scenes.
-To create an Entity it is required to provide a Mesh, which must be created first.
+An entity represents a 3D object on the scene and its pose (position and rotation).
+It is a lightweight object containing a reference to a heavier Mesh object and a transformation matrix.
+The entity's transform is a 3D affine matrix (therefore 3x4) describing the entity's local-to-scene coordinate frame transform.
+The coordinate system in RGL is right-handed: the positive x and y axes point right and up, and the negative z-axis points forward. Positive rotation is counterclockwise about the axis of rotation.
+When created, an entity is bound to a scene. An entity cannot be unbound from a scene or bound to multiple scenes.
+In order to create an entity, it is required to provide a Mesh, which must be created first.
```
Comment (prybicki) [inline docs/Usage.md:20]: Hmm

File: docs/Usage.md [hunk 6]
```diff
@@ -1,34 +1,34 @@
...
 Entities can share the same mesh.
 
 ### Scene
 
-Scene represents 'a place' where the raytracing occurs.
-User is expected to fill a scene with entities before performing ray-tracing.
+A scene represents 'a place' where the raytracing occurs.
+The user is expected to fill a scene with entities before performing raytracing.
```
Comment (prybicki) [inline docs/Usage.md:26]: Ditto.

**Review by prybicki — APPROVED** [review]
Please double check / fix items discussed offline and merge.

**Pull request merged**